server2/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:18.04
```

Comment:
please add a topline comment saying what this docker container will do in general, a high level summary

Reply:
Done.